R/create_episode_file.R

```diff
@@ -31,6 +31,9 @@ create_episode_file <- function(
     sc_client = read_file(get_sc_client_lookup_path(year)),
     write_to_disk = TRUE,
     anon_chi_out = TRUE) {
+  sc_client <- sc_client %>%
+    dplyr::select(-sc_send_lca)
+
```

Comment:
These lines are not needed here and will cause duplicates. See https://github.com/Public-Health-Scotland/source-linkage-files/pull/881/files